packages/manager/modules/pci/src/projects/project/storages/databases/components/flavors-list/style.scss

```diff
@@ -0,0 +1,28 @@
+@import '@ovh-ux/ui-kit/dist/scss/tokens/_colors';
+pci-projects-project-database-flavors-list {
+  tr.table-single-select-row {
+    td {
+      cursor: pointer !important;
+    }
+
+    &:hover,
+    &:active,
+    &:focus {
+      background-color: $p-050;
+    }
+
+    &.disabled {
+      opacity: 0.5;
+      td {
+        cursor: not-allowed !important;
+      }
+    }
+
+    &.selected {
+      td {
+        font-weight: 800 !important;
```

Comment:
```suggestion
        @extend .font-weight-bold;
```

Reply:
Done here: https://github.com/ovh/manager/pull/8392/commits/a60a8d7abd1bab5131a47f01a92f7d710645a860
I had to add `bootstrap` imports for it to work:
```
@import 'bootstrap4/scss/_functions.scss';
@import 'bootstrap4/scss/_variables.scss';
@import 'bootstrap4/scss/_mixins.scss';
@import 'bootstrap4/scss/_utilities.scss';
```